snocker/app/dockerfile

```diff
@@ -0,0 +1,101 @@
+############################################################
```

Comment:
By convention, the file name should be Dockerfile (capital D).

Reply:
I think it works either way. I am not wed to any particular convention. Feel free to change/